worker/appm/conversion/starupseq_detector.go

```diff
@@ -0,0 +1,99 @@
+package conversion
```

Comment:
Here I think we should implement a separate algorithm.  not in the conversion package.